app/models/repository.rb

```diff
@@ -206,8 +206,19 @@ def self.create_or_update!(repo)
         digest = ""
       end
 
-      Tag.create!(name: tag, repository: repository, author: portus, digest: digest, image_id: id)
-      logger.tagged("catalog") { logger.info "Created the tag '#{tag}'." }
+      if current_tag = repository.tags.find_by_name(tag)
```

Comment:
When I push a new version of `mynamespace/myimage:latest`, I don't see it appear in the "recent activities", nor do I see the digest id being updated. Maybe I'm missing something?